yarn.lock

```diff
@@ -6164,9 +6164,9 @@ __metadata:
   linkType: hard
 
 "caniuse-lite@npm:^1.0.0, caniuse-lite@npm:^1.0.30000981, caniuse-lite@npm:^1.0.30001109, caniuse-lite@npm:^1.0.30001125, caniuse-lite@npm:^1.0.30001503":
-  version: 1.0.30001513
```

Comment:
you should revert the changes to this file by running: `git checkout develop -- yarn.lock`